native/cocos/network/WebSocketServer.cpp

```diff
@@ -390,7 +390,10 @@ WebSocketServerConnection::~WebSocketServerConnection() {
 
 bool WebSocketServerConnection::send(std::shared_ptr<DataFrame> data) {
     _sendQueue.emplace_back(data);
-    onDrainData();
+    if (!_wsi || _closed || _readyState == ReadyState::CLOSING) {
+        return false;
+    }
+    lws_callback_on_writable(_wsi);
```

Comment:
These steps are already included in `onDrainData()`. 

The purpose of this commit appears to be to continue calling `lws_callback_on_writable` after `onDrainData` returns an error code?

Reply:
onServerWritable call onDrainData
sendBinaryAsync call onDrainData : 
The current time may be server can not writeable, if we send data will trigger error.
I think onDrainData should only need one entrance.

Reply:
Can you describe in detail how your error occurred? Or provide a demo and we will try to reproduce it.

Reply:
on web platform use WebSocket as client, run on browser.
on android native platform, use websocketserver as server, send data to client high frequency.
sometime it will disconnect and web log show error:
 'ws://XXXX/' failed: A server must not mask any frames that it sends to the client. 

the problem is obvious, we send data must at server writeable status, if server cann't writeable we send data it appear error.

Reply:
Could you send me a simple and reproducible demo, I need to further check the error message.